### DISCUSSION TITLE
File System API does not use standard Function Signature

### DISCUSSION THREAD

**Discussion by khrome83** [vercel/community#73] [Help] [answered]
What signature does the File System API use for it's handlers? 

When trying to use `res.status(200).send('something')` it errors out and tells me that `res.status` and `res.send` are not functions. 

Additionally, I had to make the `.js file a `.mjs` file to get it to process with export. Using `type: module` in the package.json was not respected. 

To get a response to return, I had to use - 

```
res.statusCode = 200;
res.setHeader("content-type", "text/html; charset=UTF-8")
res.end("<div>something</div");
```

What is the function model that uses this type of return signature? The nice features normally in a vercel function do not seem available. 

https://vercel.com/docs/runtimes#official-runtimes/node-js

Is there a way to enable the Node.js Helpers?

**Comment by leerob** [comment]
Could you share what you're trying to build with the File System API?

**Comment by Pieparker** [comment, marked as answer]
This conversation appears to have gone quiet. To keep discussions on topic and to prevent the resurrection of old topics, this discussion has been closed. If issues persist or a future reader stumbles upon this discussion and is exhibiting the same problem, please create a new discussion.
